src/chapter_1.md

```diff
@@ -1,9 +1,9 @@
 # Chapter 1: Hello, You!
 
-In the previous chapter we wrote a greet function and _then_ wrote the tests. 
+In the previous chapter we wrote a greet function and _then_ wrote the tests.
 
 ```rust
-{{#include ./chapter_1_examples/hello_you/src/lib.rs}}
+{{#include ./chapter_0_examples/hello_world_with_tests/src/main.rs}}
```

Comment:
I couldn't get this code that is referenced to run. I also realized that this is referring back to chapter 0, so I updated to that.